tsconfig.json

```diff
@@ -1,31 +1,19 @@
 {
   "ts-node": {
-    "compiler": "ttypescript",
-    "moduleTypes": {
-      "webpack/*": "cjs",
-      // Globs are also supported with the same behavior as tsconfig "include"
-      "webpack/**/*": "cjs"
-    },
-    "esm": true
+    "experimentalSpecifierResolution": "node"
   },
   "compilerOptions": {
-    "plugins": [
-      {
-        "transform": "@zoltu/typescript-transformer-append-js-extension/output/index.js",
-        "after": true
-      }
-    ],
-
     "target": "ES2022",
-    "module": "ES2022",
+    "module": "ESNext",
```

Comment:
I don't really like targeting a floating module spec like this, but that's more of an opinion than a need to change. Might just be the "big corporation developer" mindset coming out a bit.

Reply:
Looks like `ESNext` is currently equal to `ES2022`, until ESMA produces a new spec or something? I didn't remember why I changed this so it is okay to be reverted.